compose_to_rancher/__init__.py

```diff
@@ -56,5 +73,27 @@ def main():
     output_file_path = args.output
     read_write_compose(file_path, output_file_path)
 
+#######################
+# Utility Functions
+#######################
+
+def _add_or_merge(dictionary, key, values):
```

Comment:
Clever function! 

Could you rename it to `add_or_merge_dict` without the trailing `_`?

I think this should be a top level function, not an "internal use" one.